.pre-commit-config.yaml

```
@@ -18,3 +18,26 @@ repos:
     hooks:
       - id: prettier
         types_or: [css, scss, javascript, ts, tsx, json, yaml]
+  - repo: https://github.com/pre-commit/mirrors-eslint
+    rev: v8.8.0
+    hooks:
+      - id: eslint
```

We’ll need to make sure we’re processing TypeScript files too, and use the same options as the package.json (max-warnings shouldn’t be defined there):

```suggestion
      - id: eslint
        types: [file]
        files: \.(js|ts|tsx)$
        args: [--report-unused-disable-directives]
```

---

Thanks for this, I probably wouldn't have ever caught this :)